<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1410</width>
    <height>1022</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>全自动雌雄蚕蛹分选机 V2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="capimage1">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>90</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>采集图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_station1_totalnum">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>890</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_station1_passnum">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>790</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_station1_failnum">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>840</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1061</x>
      <y>840</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>雄蛹数量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>测试总数</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="cam_win_1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>651</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ck_edit">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>510</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>510</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>串口端口1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ckopen_Button">
    <property name="geometry">
     <rect>
      <x>1320</x>
      <y>510</y>
      <width>60</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QLabel" name="ckStatus_label">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>620</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>790</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>雌蛹数量</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>380</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>374</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>阈值参数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit1">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>420</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>420</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>面积参数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>460</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>80</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>460</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>辅助系数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>610</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>串口状态:</string>
    </property>
   </widget>
   <widget class="QLabel" name="cam_win_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>651</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>710</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>当前检测结果：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>720</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>逻辑复位</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>250</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清零</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gif">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>1190</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>320</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>延时参数</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>340</y>
      <width>120</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>3500</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>560</y>
      <width>141</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>串口端口2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ck_edit2">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>560</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1410</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
